runtime/src/main/java/com/datastax/oss/quarkus/runtime/reactive/MutinyReactiveResultSet.java

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.oss.quarkus.runtime.reactive;
+
+import com.datastax.dse.driver.api.core.cql.reactive.ReactiveResultSet;
+import com.datastax.dse.driver.api.core.cql.reactive.ReactiveRow;
+import com.datastax.oss.driver.api.core.cql.ColumnDefinitions;
+import com.datastax.oss.driver.api.core.cql.ExecutionInfo;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import io.smallrye.mutiny.Multi;
+import io.smallrye.mutiny.Uni;
+import io.smallrye.mutiny.groups.MultiBroadcast;
+import io.smallrye.mutiny.groups.MultiCollect;
+import io.smallrye.mutiny.groups.MultiConvert;
+import io.smallrye.mutiny.groups.MultiGroup;
+import io.smallrye.mutiny.groups.MultiOnCompletion;
+import io.smallrye.mutiny.groups.MultiOnEvent;
+import io.smallrye.mutiny.groups.MultiOnFailure;
+import io.smallrye.mutiny.groups.MultiOnItem;
+import io.smallrye.mutiny.groups.MultiOverflow;
+import io.smallrye.mutiny.groups.MultiSubscribe;
+import io.smallrye.mutiny.groups.MultiTransform;
+import java.util.concurrent.Executor;
+import java.util.function.Function;
+import java.util.function.Predicate;
+import org.reactivestreams.Publisher;
+import org.reactivestreams.Subscriber;
+
+public class MutinyReactiveResultSet implements Multi<ReactiveRow>, ReactiveResultSet {
+
+  private final ReactiveResultSet reactiveResultSet;
+  private final Multi<ReactiveRow> multi;
+
+  public MutinyReactiveResultSet(ReactiveResultSet reactiveResultSet) {
+
+    this.reactiveResultSet = reactiveResultSet;
+    this.multi = Wrappers.toMulti(reactiveResultSet);
+  }
+
+  @NonNull
+  @Override
+  public Publisher<? extends ColumnDefinitions> getColumnDefinitions() {
+    return reactiveResultSet.getColumnDefinitions();
+  }
+
+  @NonNull
+  @Override
+  public Publisher<? extends ExecutionInfo> getExecutionInfos() {
```

Comment:
Should return `Multi<ExecutionInfo>`.

Reply:
Same as: https://github.com/datastax/cassandra-quarkus/pull/11#discussion_r425729715